backend/sprintone/src/main/java/com/project/omega/helper/Constant.java

```diff
@@ -0,0 +1,12 @@
+package com.project.omega.helper;
+
+public interface Constant {
+    public static final String ERROR_USER_EXISTS = "User already exists | email: ";
+    public static final String ERROR_NO_RECORDS = "No user records found.";
+    public static final String ERROR_NO_RECORDS = "No records found.";
+    public static final String ERROR_USER_NOT_FOUND = "User not found | id: ";
+    public static final String ERROR_NO_PRODUCTS = "No products found.";
+    public static final String ERROR_PRODUCT_NOT_FOUND = "Product not found | id: ";
+    public static final String ERROR_INDUSTRY_NOT_FOUND = "Industry not found | id: ";
+    public static final String ERROR_INDUSTRY_EXISTS = "Industry already exists | id: ";
```

Comment:
This is repeated in `Backend/OmegaSpring`. Also, name the class/interface a bit more appropriately